src/test/Runner.test.ts

```diff
@@ -57,6 +60,52 @@ describe('Runner', () => {
       expect(await runner.run()).toEqual(false);
       expect(runner.retriedCount).toEqual(3);
     });
+
+    test('it generates output profile', async () => {
+      await copy();
+      const outputProfile = resolve('profile.json');
+      const runner = new Runner({
+        retry: 2,
+        config: 'backstop.json',
+        referenceCommand: 'not_existing_command',
+        command: 'not_existing_command',
+        outputProfile,
+        rootDir: resolve('backstop/failed'),
+      });
+      await runner.run();
+      expect(JSON.parse(fs.readFileSync(outputProfile).toString())).toEqual([
+        {
+          label: 'backstop.run',
```

Comment:
I added a `backstop.` prefix for future expansion, but it may be a bit redundant.

Reply:
I think `run` is enough, because this software is a backstop runner.